sql/moz-fx-data-shared-prod/monitoring_derived/bigquery_table_storage_timeline_daily_v1/metadata.yaml

```diff
@@ -1,4 +1,4 @@
-friendly_name: Bigquery Query Tables Storage Timeline Daily
+friendly_name: Bigquery Tables Storage Timeline Daily
 description: The daily timeline in table storage changes, partitioned by day.
```

Comment:
Maybe could expand here what we mean by storage changes? Does the table contain diff in comparison to previous day or is it total storage size in GB? Or are we just talking about less detail information like "a specific partition was updated on x"?

Reply:
There is ongoing work to refine this table.  I will add this feedback to it.